Cargo.toml

```diff
@@ -32,3 +32,6 @@ wasm-bindgen = { version = "0.2", optional = true }
 
 [dev-dependencies]
 wasm-bindgen-test = "0.2"
+
+[target.'cfg(feature="wasm-bindgen")'.dependencies]
+web-sys = {version="0.3.19", optional = true, features=['Window', 'Performance', 'PerformanceTiming']}
```

Comment:
I do not think we can check enabled features in a `target.cfg(...)`. I think we can only check target-related attributes: https://doc.rust-lang.org/cargo/reference/config.html

Reply:
We probably want something like this:
```toml
[features]
now = ["time"]
wasm-bindgen = ["wasm_bindgen", "web_sys"]

[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
time = { version = "0.1", optional = true }

[target.'cfg(target_arch = "wasm32")'.dependencies]
stdweb = { version = "0.4", optional = true }
wasm_bindgen = { package = "wasm-bindgen", version = "0.2", optional = true }
web_sys = { package = "web-sys", version = "0.3", optional = true }
```